Assets/Placenote/Toolkits/SpatialCapture/CapturedPlaneUtility.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace UnityEngine.XR.iOS.SpatialCapture
```

Comment:
same change to namespace as shown in previous comment

Reply:
Done